packages/blade/src/components/List/_decisions/decisions.md

```diff
@@ -0,0 +1,305 @@
+# List <!-- omit in toc -->
+Lists display a set of related items that are composed of text/links. Each list item begins with a bullet or a number.
+
+<img src="./list-thumbnail.png" width="380" />
+
+- [Design](#design)
+  - [Unordered List](#unordered-list)
+    - [Unordered List with 3 levels of nesting](#unordered-list-with-3-levels-of-nesting)
+    - [Unordered List with custom icons for bullets](#unordered-list-with-custom-icons-for-bullets)
+    - [Unordered List with inline Link in items](#unordered-list-with-inline-link-in-items)
+  - [Ordered List](#ordered-list)
+    - [Ordered List with 3 levels of nesting](#ordered-list-with-3-levels-of-nesting)
+    - [Ordered List with 3 levels of nesting and _filled background_ numbers](#ordered-list-with-3-levels-of-nesting-and-filled-background-numbers)
+    - [Ordered List with inline Link in items](#ordered-list-with-inline-link-in-items)
+- [API](#api)
+  - [List Component](#list-component)
+  - [ListItem Component](#listitem-component)
+  - [ListItemLink Component](#listitemlink-component)
+  - [Sample Usage](#sample-usage)
+  - [Alternative APIs](#alternative-apis)
+    - [Hierarchial with `variant` as `'ordered' | 'ordered-filled' | 'unordered'`](#hierarchial-with-variant-as-ordered--ordered-filled--unordered)
+    - [Originally proposed API](#originally-proposed-api)
+    - [Hierarchial with separate OrderedList \& UnorderedList Components](#hierarchial-with-separate-orderedlist--unorderedlist-components)
+    - [Hierarchial without explicit nested `List` grouping](#hierarchial-without-explicit-nested-list-grouping)
+    - [Hierarchial with a `nested` prop](#hierarchial-with-a-nested-prop)
+    - [Non-Hierarchial with a `level` prop](#non-hierarchial-with-a-level-prop)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+
+## Design
+- [Figma - List](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=23205%3A446859&t=dT849yOxgX5WHgby-4)
+
+### Unordered List
+#### Unordered List with 3 levels of nesting
+
+<img src="./unordered-list.png" width="380" />
+
+#### Unordered List with custom icons for bullets
+
+<img src="./unordered-list-icon.png" width="380" />
+
+#### Unordered List with inline Link in items
+
+<img src="./unordered-list-link.png" width="380" />
+
+### Ordered List
+#### Ordered List with 3 levels of nesting
+
+<img src="./ordered-list.png" width="380" />
+
+#### Ordered List with 3 levels of nesting and _filled background_ numbers
+
+<img src="./ordered-list-filled.png" width="380" />
+
+#### Ordered List with inline Link in items
+
+<img src="./ordered-list-link.png" width="380" />
+
+
+## API
+
+### List Component
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| children | `ListItem` | `undefined` | List items to render. | ✅ |
+| variant | `ordered`, `ordered-filled`, `unordered` | `unordered` | Sets the list as ordered or unordered. |  |
```

Comment:
In nested ordered list, how would the user specify the indexing pattern (similar to the screenshot above)?
- 1, 2, 3
- a, b, c
- i, ii, iii

Reply:
Level 1 will be 1,2,3
Level 2 will be a,b,c
Level 3 will be i,ii,iii

Reply:
Maybe we can specify that here

Reply:
Done